include/swift/ClangImporter/ClangImporter.h

```diff
@@ -38,10 +38,11 @@ namespace clang {
   class Sema;
   class TargetInfo;
   class VisibleDeclConsumer;
+  class DeclarationName;
 }
 
 namespace swift {
-
+namespace version { class Version; }
```

Comment:
I removed the version dependency all together, because I cannot foresee any version input that is different from current one.